usermembership/tool/src/webapp/usermembership/userlist.jsp

```diff
@@ -148,15 +148,19 @@
 		                <h:outputText value="#{msgs.user_created_on}"/>
 		            </t:commandSortHeader>
 		        </f:facet>
-				<h:outputText value="#{row.createdOn}"/>
+				<h:outputText value="#{row.createdOn}">
+					<f:convertDateTime pattern="#{msgs.date_format}"/>
+				</h:outputText>
 			</h:column>
 			<h:column id="modifiedOn">
 				<f:facet name="header">
 		            <t:commandSortHeader columnName="modifiedOn" immediate="true" arrow="true">
 		                <h:outputText value="#{msgs.user_modified_on}"/>
 		            </t:commandSortHeader>
 		        </f:facet>
-				<h:outputText value="#{row.modifiedOn}"/>
+				<h:outputText value="#{row.modifiedOn}">
```

Comment:
I'm rusty on JSF, what output do we get when `modifiedOn` is `null`?

Reply:
Just tested this and it copes fine with `null` values, it just outputs an empty string.